src/DataObjects/Moderation/Moderation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace OpenAI\DataObjects\Moderation;
+
+use OpenAI\Contracts\DataObject;
+
+final class Moderation implements DataObject
```

Comment:
This class should be name `CreateResponse` and it should implement the `Response` interface.